packages/@aws-cdk/aws-rds/README.md

```diff
@@ -2,6 +2,14 @@
 
 The `aws-cdk-rds` package contains Constructs for setting up RDS instances.
 
+> Note: this package is currently quite limited, as the CDK team is focusing
+> on other use cases first. If your use case is not in the list below, there is
+> no use coming in to the channel to ask "how do I..." because it won't be
```

Comment:
Passive aggressive?

Reply:
Getting fed up with the same questions over and over again. But yes, this is probably not the best message to put there. Suggestions?

Reply:
+1 you can just point to the escape hatches topic and mention that this is not complete yet.